invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMemberBulkActions.js

```diff
@@ -156,13 +156,13 @@ class ManagerMemberBulkActionsCmp extends Component {
     const currentActionRender =
       this.currentAction?.renderOnActive && this.currentAction.renderOnActive();
 
-    const { text: currentActionText, actionParam } = this.currentAction;
+    const currentActionText = this.currentAction?.text;
 
     const actionDisabled =
       loading ||
       selectedCount === 0 ||
       // eslint-disable-next-line react/destructuring-assignment
-      !this.state[actionParam] === undefined;
+      !this.state[this.currentAction.actionParam] === undefined;
```

Comment:
I had this issue when fixing the linting warnings.
I managed to keep the destructuring by checking the variable before using it, so maybe it could be an option.

Reply:
this is not a variable but a getter function, that is why there is a problem with destructuring, see the answer here on why this is happening
https://stackoverflow.com/questions/69580147/destructuring-losing-getter-definition